### ISSUE TITLE
REPORTS: UCM

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#5023]
We also need some dedicated time to figure out formatting....loans printed with the old coldfusion reporter have a better look over the new, e.g., 
old: [V-233_McCain_MammalogyClass_Fall2021.pdf](https://github.com/ArctosDB/arctos/files/9500594/V-233_McCain_MammalogyClass_Fall2021.pdf) vs.
new: [reporter.pdf](https://github.com/ArctosDB/arctos/files/9500621/reporter.pdf) and https://arctos.database.museum/Reports/reporter.cfm?action=view&report_id=3&&transaction_id=21127839 (the UCM-loan_invoice doesn't generate the pdf for some reason)

_Originally posted by @ebraker in https://github.com/ArctosDB/arctos/issues/4986#issuecomment-1238637174_

**Comment by ebraker** [comment]
Ok, so hopefully some low-hanging fruit:
1. reduce font size
2. use san sarif fonts

Existing Loan Header that we love and want to replicate:
![image](https://user-images.githubusercontent.com/11336485/189445873-91ad35c9-89a2-4d47-b735-864737523cfb.png)

versus new reporter Loan Header that still needs tweaking (font so large it runs onto two pages - we need a 1-pg document):
![image](https://user-images.githubusercontent.com/11336485/189445963-70eaa579-2db3-497b-9a78-e0d3cdd7efd0.png)

Existing Loan Invoice that we love and want to replicate:
![image](https://user-images.githubusercontent.com/11336485/189447035-c6f53ad3-cc58-4186-bcf2-5dd46245f6b6.png)


versus new reporter Loan Invoice (blank pdf...unable to view):
![image](https://user-images.githubusercontent.com/11336485/189446441-83e5dab2-f232-4b4c-bc50-f262f949572e.png)

Info to replicate:
- Loan: https://arctos.database.museum/Loan.cfm?action=editLoan&transaction_id=21127839
- UCM_Loan_Header: http://reports.arctos.database.museum/reporter/reporter.cfm?action=edit&report_id=5601091&auth_key=74D52E41-3AB7-403C-92F49C64B7937B4A
- UCM_Loan_Invoice http://reports.arctos.database.museum/reporter/reporter.cfm?action=edit&report_id=5601092&auth_key=74D52E41-3AB7-403C-92F49C64B7937B4A

**Comment by ebraker** [comment]
@dustymc @lkvoong let me know if I should resubmit this as a new report template request

**Comment by lkvoong** [comment]
@ebraker 
Hi Emily,
Sorry, I just moved the reports to production.  Now that it's on production, you can try printing them yourself.  I'm attaching the pdfs of the reports for Loan V-233.  For the Loan Invoice, I haven't been able to successfully get a version with a repeatable column header to work.  Also, I've set the Collect Date to not wrap, which does not work for some records with no date recorded (e.g. "no date recorded - before 8 November 1966"), so I can update it to wrap for those records.  Please review and let me know what other changes should be made. Thanks!

[UCM-Loan-Header-V-233.pdf](https://github.com/ArctosDB/arctos/files/9891315/UCM-Loan-Header-V-233.pdf)

[UCM-Loan-Invoice-V-233.pdf](https://github.com/ArctosDB/arctos/files/9891317/UCM-Loan-Invoice-V-233.pdf)

**Comment by ebraker** [comment]
@lkvoong Thank you!!! These look fantastic! 

I was able to italicize and wrap scientific names and verbatim dates since some of the subspecies names ran into sex data (feel free to edit if there is a better approach...I've never used CSS and was just clunking around).

- Repeating the loan invoice column header is not urgently necessary, but I'll definitely add it in if you end up figuring it out
- Is there a way to get the invoice header loan number to sit on the same line as the word "loan"?
![image](https://user-images.githubusercontent.com/11336485/198739340-cf59362a-9b69-4566-bf77-81693423b7e3.png)
- In opening the V-233 invoice, there are a few pages where there is some text overflow to the next page and makes the table wonky, e.g.
![image](https://user-images.githubusercontent.com/11336485/198739245-44cb30b4-a56d-449f-a2e4-062a85d48bc6.png)

- Last but not least is there a way to auto-incorporate the report title during download? Right now when I print->save as pdf, each report title is defaulted to "reporter" in the 'save as' field. I'm usually downloading both a loan header and invoice in the same span of minutes then combining the pdfs in Adobe Acrobat, so if I don't rename, I overwrite my previously downloaded report (this is a lazy ask, so it's not a huge deal if it can't be done!).

Thank you so much. I was dreading this transition after putting in so much time into cold fusion. What a huge relief! I already printed off a loan with the new templates.
